l10n_fr_ecotaxe_sale/views/sale_view.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+   Copyright (C) 2015-2020 Akretion (http://www.akretion.com/)
+   @author: Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+-->
+<odoo>
+    <record id="view_order_form" model="ir.ui.view">
+        <field name="name">l10n_fr_ecotaxe.sale.line.form</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form" />
+        <field name="arch" type="xml">
+            <!-- <field name="amount_untaxed"> -->
```

Comment:
```suggestion
```
Better to remove this comment if it's not used

Reply:
I kept it to find better place to put this field on. Previously it was under amount_untaxed but there is no such field anymore. I guess it should go into total price section under so lines. Comment will be removed as soon as I'll move this field into better place.